src/components/Header/Header.jsx

```diff
@@ -1,18 +1,49 @@
+import { useEffect, useState, useCallback } from "react";
 import { Link } from "react-router-dom";
+import { debounce } from "lodash";
+import { getProductByText } from "api/Api";
```

Comment:
```suggestion
import { getProductByText } from "api";
```
через index файлы делаем импорты